app/lib/actions/katello/content_view/capsule_sync.rb

```diff
@@ -9,7 +9,7 @@ def humanized_name
         def plan(content_view, environment)
           sequence do
             concurrence do
-              smart_proxies = SmartProxy.with_environment(environment)
+              smart_proxies = SmartProxy.unscoped.authorized(:view_capsule_content).with_environment(environment)
```

Comment:
I think these should all be:

```suggestion
              smart_proxies = SmartProxy.unscoped.authorized(:manage_capsule_content).with_environment(environment)
```

Reply:
That specifically didn't work so I had to tweak it a little while using manage_capsule_content..